scipy-1.0/paper.tex

```diff
@@ -258,6 +262,20 @@ \subsubsection*{LowLevelCallable}
 
 \subsection*{Cython bindings for BLAS, LAPACK and special}
 
+When writing algorithms or wrappers in Cython, it is often convenient to call directly into the libraries that SciPy wraps rather than routing the call back through the Python interpreter.
+SciPy currently includes Cython wrappers for many BLAS and LAPACK routines and for many of the special functions provided in the \texttt{scipy.{\allowbreak}special} submodule.
+These wrappers are available in the modules \texttt{scipy.{\allowbreak}linalg.{\allowbreak}cython\_blas}, \texttt{scipy.{\allowbreak}linalg.{\allowbreak}cython\_lapack}, and \texttt{scipy.{\allowbreak}special.{\allowbreak}cython\_special} respectively.
+The low-level interfaces for Cython can also be used by third party libraries to get direct access to the functions in these libraries while avoiding the overhead of Python function calls.
```

Comment:
Because many readers won't know what this means, maybe add something like "- this can give performance gains of one or two orders of magnitude for many use cases".

Reply:
Thanks for this, I've added text to that effect.